README.md

```diff
@@ -1,80 +1,81 @@
-# zindi-package
+# zindi package
 
-## Description
-
-A user-friendly ZINDI package which allow Zindians to achieve all available tasks on ZINDI Platform using this package.
+A user-friendly Zindi package which helps Zindians get things done on the Zindi platform.
 
 ## Installation
 
-Copy and Paste the instruction below in a Terminal or Jupyter Notebook.
-
 ```bash
 pip install git+https://github.com/eaedk/testing-zindi-package.git
 ```
 
+Or even:
+
+```bash
+pip install https://github.com/eaedk/testing-zindi-package/archive/master.tar.gz
+```
```

Comment:
Installing from `https://github.com/eaedk/testing-zindi-package/archive/master.tar.gz` is faster than installing from  `git+https://github.com/eaedk/testing-zindi-package.git`, since it only downloads a snapshot of the latest version, not the entire history. [See stackoverflow answer][1]

[1]: https://stackoverflow.com/a/24811490